concent_api/core/tests/test_integration_force_payment.py

```diff
@@ -584,3 +591,86 @@ def test_provider_send_force_payment_with_same_subtasks_id_concent_should_refuse
+                timestamp       = "2018-02-05 10:00:15",
+                payment_ts      = "2018-02-05 12:00:00",
+                task_to_compute = self._get_deserialized_task_to_compute(
+                    timestamp                       = "2018-02-05 10:00:00",
+                    deadline                        = "2018-02-05 10:00:10",
+                    task_id                         = '2',
+                    requestor_public_key            = self._get_encoded_requestor_public_key(),
+                    requestor_ethereum_public_key   = self._get_encoded_requestor_ethereum_public_key(),
+                )
+            ),
+            self._get_deserialized_subtask_results_accepted(
+                timestamp       = "2018-02-05 9:00:15",
+                payment_ts      = "2018-02-05 11:00:00",
+                task_to_compute = self._get_deserialized_task_to_compute(
+                    timestamp                       = "2018-02-05 9:00:00",
+                    deadline                        = "2018-02-05 9:00:10",
+                    task_id                         = '3',
+                    requestor_public_key            = self._get_encoded_requestor_public_key(),
+                    requestor_ethereum_public_key   = self._get_encoded_requestor_ethereum_public_key(),
+                )
+            )
+        ]
+        serialized_force_payment = self._get_serialized_force_payment(
+            timestamp                     = "2018-02-05 12:00:20",
+            subtask_results_accepted_list = subtask_results_accepted_list
+        )
+
+        with freeze_time("2018-02-05 12:00:20"):
+            with mock.patch('core.views.base.get_list_of_transactions', _get_requestor_valid_list_of_transactions):
+                with mock.patch('core.views.base.payment_summary', _get_payment_summary_positive):
+                    with mock.patch('core.views.base.make_payment_to_provider', _make_payment_to_provider):
+                        response_1 = self.client.post(
+                            reverse('core:send'),
+                            data                                = serialized_force_payment,
+                            content_type                        = 'application/octet-stream',
+                            HTTP_CONCENT_CLIENT_PUBLIC_KEY      = self._get_encoded_provider_public_key(),
+                            HTTP_CONCENT_OTHER_PARTY_PUBLIC_KEY = self._get_encoded_requestor_public_key(),
+                        )
+        self._test_response(
+            response_1,
+            status       = 200,
+            key          = self.PROVIDER_PRIVATE_KEY,
+            message_type = message.concents.ForcePaymentCommitted,
+            fields       = {
+                'recipient_type': message.concents.ForcePaymentCommitted.Actor.Provider,
+                'timestamp':      self._parse_iso_date_to_timestamp("2018-02-05 12:00:20"),
+                'amount_pending': Decimal('10.01'),
```

Comment:
you could make variables out of those two values -> `amount_pending`, `amount_paid` and reuse them ;)

Reply:
Done